packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2664,13 +2668,7 @@ export class ContainerRuntime
 				// stashed ops flow. The compressor is stashed with these ops already processed.
 				// That said, in idCompressorMode === "delayed", we might not serialize ID compressor, and
 				// thus we need to process all the ops.
-				if (
-					!(
-						this.skipSavedCompressorOps &&
-						(messageWithContext.message.metadata as IIdAllocationMetadata)?.savedOp ===
-							true
-					)
-				) {
+				if (!(this.skipSavedCompressorOps && messageWithContext.savedOp)) {
```

Comment:
nit: Any reason to replace the "safer" `object.savedOp === true` with just `object.savedOp`?

Reply:
IMHO, it's not worth it (we need to trust our type info somehow, otherwise we will be adding IF statements all over the place, testing that arguments to functions match declared types). I.e. if we can trust that it's undefined | boolean, then both checks are identical, and current code is smaller.

But at the same time, I'm fine to change it, so changing it. :)